### PR TITLE
Initial business architecture covering expected functionalities

### LINKED ISSUE #1: CentOps: Design CentOps Architecture
AS AN Architect
I WANT TO document and clarify the functional requirements and desired capabilities of the Central Operations System in Buerokratt
SO THAT the team can choose the appropriate technologies and solutions to ensure it meets those goals.

- [x] Produce a draft architectural design
- [x] Present to the team to solicit feedback
- [ ] Commit a design document which captures the goals of the system

### PR DESCRIPTION
**Pull request by turnerrainer**
Initial commit for CentOps business architecture

### CODE REVIEW

File: docs/technical/centops-business-architecture.doc.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+| Get a list of planned outages of all registered participants            |               *               |          |
+| Inform about a planned outage of all components of an Institution       | Institutions, Buerokratt Core |          |
+| Inform about a planned outage of a specific component of an Institution | Institutions, Buerokratt Core |          |
+| Inform about changes in planned outages                                 | Institutions, Buerokratt Core |          |
+| Update information about outages                                        |        Buerokratt Core        |          |
+| Publish information about outages                                       |        Buerokratt Core        |          |
+| Notify participants about outages                                       |        Buerokratt Core        |          |
+
+
+# Monitoring
```
Comment (guypritchard) [inline docs/technical/centops-business-architecture.doc.md]:
```suggestion
### Monitoring
```

File: docs/technical/centops-business-architecture.doc.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+## Features
+
+### Institutions
+
+*Ministries etc*
+
+| Service                                                           |  Requests from  | Priority |
+|-------------------------------------------------------------------|:---------------:|:--------:|
+| Get a list of all registered institutions                         |        *        |          |
+| Register a new institution                                        | Buerokratt Core |          |
```
Comment (guypritchard) [inline docs/technical/centops-business-architecture.doc.md]: We spoke about this yesterday but it's worth having a detailed description of the registration process (it's probably not this document maybe the technical design?)

**Review by guypritchard — APPROVED** [review]
We can continue to shape this - but this certainly gets things started, thank you.

**Review by tonistuulNG — APPROVED** [review]

File: docs/technical/centops-business-architecture.doc.md (hunk 2 above)
Comment (turnerrainer) [reply docs/technical/centops-business-architecture.doc.md]: I'll cover it here in next iteration

**Pull request merged**